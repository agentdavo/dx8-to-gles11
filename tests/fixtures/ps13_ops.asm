ps.1.3
texdp3
mul r0, v0, v1
texdp3tex t0
add r0, r0, v1
texm3x3
nop
