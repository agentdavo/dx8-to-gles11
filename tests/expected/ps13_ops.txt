TEX_ENV_COMBINE func=34478
TEX_ENV_COMBINE func=8448
TEX_ENV_COMBINE func=34478
TEX_SAMPLE stage=0
TEX_ENV_COMBINE func=260
TEX_ENV_COMBINE func=34478
TEX_ENV_COMBINE func=34478
TEX_ENV_COMBINE func=34478
